CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Next Release
   * Reset Capybara driver to Puffing Billy (used to rewrite URL requests in specs)
```

Comment:
Next release is 1.6.2 now, you can add this to your changelog and delete the next release point